assets/screenshots/amoScreenshots.csv

```diff
@@ -0,0 +1,16 @@
+autocorrectedTextChrome.png, "Autocorrected Text, Chrome"
+autocorrectedTextFirefox.png, "Autocorrected Text, Firefox"
+autocorrectedTextThunderbird.png, "Autocorrected Text, Thunderbird"
+optionsPageChrome.png, "Options Page, Chrome"
+optionsPageFirefox.png, "Options Page, Firefox"
+optionsPageThunderbird.png, "Options Page, Thunderbird"
+textTransformOptionsChrome.png, "Text Transform options, Light theme, Chrome"
+textTransformOptionsDarkChrome.png, "Text Transform options, Dark theme, Chrome"
+textTransformOptionsDarkFirefox.png, "Text Transform options, Dark theme, Firefox"
+textTransformOptionsFirefox.png, "Text Transform options, Light theme, Firefox"
+textTransformOptionsLivePreviewDarkFirefox.png, "Text Transform options with Live preview, Dark theme, Firefox"
+textTransformOptionsLivePreviewFirefox.png, "Text Transform options with Live preview, Light theme, Firefox"
+textTransformOptionsLivePreviewDarkThunderbird.png, "Text Transform options with Live preview, Dark theme, Thunderbird"
+textTransformOptionsLivePreviewThunderbird.png, "Text Transform options with Live preview, Light theme, Thunderbird"
```

Comment:
As for the text here, don’t be so objective… :stuck_out_tongue_winking_eye: 

This is no image description that just needs to describe what you can already see, but should be some marketing thing, some “fluffy” words. Again [see our Offline QR Code example](https://github.com/rugk/offline-qr-code/blob/master/assets/texts/en/amoScreenshots.csv).
It’s basically about highlighting/presenting nice features and at best also describing boring pages like the options page in a positive way that makes users think “This is cool/special/I want that.”, you see… Try to invoke emotions, (as far as that’s possible) but of course also stay honest and exact. :smile: It should also be quite brief, as it should be easy and fast to read.

E.g. for the settings:
> Disable some features you do not use or just make use of all of them. You can adjust the style and behaviour as you like!

etc. etc.

(If the image only differs in being dark or light of course just use the same description, we’ll likely only use/upload one version as said… it makes no sense to display all of them in all styles.)

If you are out of ideas, I can also look at the other’s and think of some descriptions that would fit, if you want. :wink:

Reply:
> If the image only differs in being dark or light of course just use the same description, we’ll likely only use/upload one version as said… it makes no sense to display all of them in all styles.

OK, I was only following your suggestion in https://github.com/rugk/unicodify/issues/25#issuecomment-841317348.

> If you are out of ideas, I can also look at the other’s and think of some descriptions that would fit, if you want. 😉

Please feel free to suggest other descriptions! There are of course only four unique images (without the browser and theme variations), so what about these descriptions:

1. Autocorrect more than 85 Unicode symbols as you type.
2. Transform selected text to more than 20 Unicode fonts or change the casing.
3. Show a live preview of all the text transform options.
4. Disable any features you do not want or just make use of them all. You can adjust the style and behavior as you like!

You are definitely much better at generating/writing them! 😀

Reply:
Well…ł you are not bad either and I like them quite much.

As for:
```diff
- Show a live preview of all the text transform options.
+ Show a live preview of all the text transform options directly in the context menu.
```

(to highlight where it is shown)

Anything else sounds very good as far as I see.

Ah only this:
```diff
- Transform selected text to more than 20 Unicode fonts or change the casing.
+ Transform the selected text to more than 20 Unicode fonts or change the casing.
```

(I guess that’s grammatically more correct :wink: )